includes/widgets/image.php

```diff
@@ -600,6 +600,37 @@ private function get_caption( $settings ) {
 		return $caption;
 	}
 
+	public function set_svg_size( $image_data, $attachment_id, $size ) {
+		$image_sizes = Group_Control_Image_Size::get_all_image_sizes();
+
+		$image_data['1'] = $image_sizes[ $size ][ 'width' ];
+		$image_data['2'] = $image_sizes[ $size ][ 'height' ];
+
+		return $image_data;
+	}
+
+	private function handle_svg_image( $action, $settings ) {
+		$is_custom_size= 'custom' === $settings[ 'image_size' ];
+		$callback = 'set_svg_size';
+		$filter = 'wp_get_attachment_image_src';
+
+		if ( 'before_render' === $action ) {
+			if ( $is_custom_size ) {
+				$custom_dimensions = $settings[ 'image_custom_dimension' ];
+
+				add_image_size( 'custom', $custom_dimensions[ 'width' ], $custom_dimensions[ 'height' ] );
+			}
+
+			add_filter( $filter, [ $this, $callback ], 10, 4 );
```

Comment:
Consider use plain strings instead of variables.

Reply:
Fixed.

Reply:
Fixed.